applications/serial_lte_modem/sysbuild/b0/prj.conf

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+CONFIG_IS_SECURE_BOOTLOADER=y
+CONFIG_MULTITHREADING=n
+CONFIG_GPIO=n
+CONFIG_ARM_MPU=n
+CONFIG_TICKLESS_KERNEL=n
+CONFIG_ERRNO=n
+CONFIG_FPROTECT=y
+CONFIG_SECURE_BOOT_CRYPTO=y
+CONFIG_SECURE_BOOT_VALIDATION=y
+CONFIG_SECURE_BOOT_STORAGE=y
+CONFIG_BL_ROT_VERIFY_EXT_API_ENABLED=y
+CONFIG_BL_SHA256_EXT_API_ENABLED=y
+CONFIG_BL_SECP256R1_EXT_API_ENABLED=y
+CONFIG_BL_VALIDATE_FW_EXT_API_ENABLED=y
+CONFIG_EXT_API_PROVIDE_EXT_API_ENABLED=y
+CONFIG_NRFX_NVMC=y
+CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_TIMEOUT_64BIT=n
```

Comment:
Where do these all come from?

Reply:
From NSIB bootloader: https://github.com/nrfconnect/sdk-nrf/blob/main/samples/bootloader/prj.conf.

Sysbuild does not support for changing only few values, so we have to have them all if we need some of these.